PredicateKit/Predicate.swift

```diff
@@ -370,6 +370,10 @@ public func == <E: Expression, T: Equatable & Primitive> (lhs: E, rhs: T) -> Pre
   .comparison(.init(lhs, .equal, rhs))
 }
 
+public func != <E: Expression, T: Equatable & Primitive> (lhs: E, rhs: T) -> Predicate<E.Root> where E.Value == T {
+  .not(.comparison(.init(lhs, .equal, rhs)))
```

Comment:
I think it might be preferable to add `.notEqual` in `ComparisonOperator` and do something like the following:

```suggestion
  .comparison(.init(lhs, .notEqual, rhs))
```

What do you think?

Reply:
Agreed. I'll fix it.